src/ThreeJsPanel.ts

```diff
@@ -355,23 +407,27 @@ export class ThreeJsPanel {
         this.replaceCamera(this.orthographicCameraX);
         this.replaceControls(this.orthoControlsX);
         this.axisHelperObject.rotation.set(0, Math.PI * 0.5, 0);
+        this.setOrthoScaleBarVisible(true);
```

Comment:
always visible?  should we have some api to show/hide it? and eventually the inevitable button..